app/meta.go

```diff
@@ -0,0 +1,30 @@
+package app
+
+import (
+	"strconv"
+
+	"fyne.io/fyne/v2"
+	intapp "fyne.io/fyne/v2/internal/app"
+)
+
+var (
+	meta fyne.AppMetadata
+)
+
+func init() {
+	build, err := strconv.Atoi(intapp.MetaBuild)
```

Comment:
Is it not possible to inject an integer directly instead, or does it only support strings?

Reply:
Yeah -X only supports strings